voice/bxml/callbacks/transferComplete.md

```diff
@@ -80,6 +82,7 @@ POST http://[External server URL]
 	"transferCallerId" : "+15551115555",
 	"transferTo"       : "+15556667777",
 	"startTime"        : "2019-07-31T13:13:34.859Z",
+	"answerTime"       : "2019-06-20T15:54:25.432Z",
```

Comment:
Should happen in the same day.

Reply:
In this example particularly the result was "busy", so it shouldn't really have an answer time, but I don't know if it will return null or not at all.